samples/private-endpoint/README.md

```diff
@@ -0,0 +1,68 @@
+---
+name: ChatGPT + Enterprise data + private endpoints
+description: Disable public access to this Chat App with environment variables.
+languages:
+- python
+- typescript
+- bicep
+- azdeveloper
+products:
+- azure-openai
+- azure-cognitive-search
+- azure-app-service
+- azure
+page_type: sample
+urlFragment: azure-search-openai-demo-private-access
+---
+# Deploying with private access
+
```

Comment:
Possible first paragraph for this one:

"The [azure-search-openai-demo](https://github.com/Azure-Samples/azure-search-openai-demo/) project can set up a full RAG chat app on Azure AI Search and OpenAI so that you can chat on custom data, like internal enterprise data or domain-specific knowledge sets. For full instructions on setting up the project, consult the [main README], and then return here for detailed instructions on configuring private endpoints."